docs/src/design/decoder/main.md

```diff
@@ -134,14 +136,14 @@ We denote the running product column used to keep track of hash chiplet state as
 To initiate a 2-to-1 hash of $8$ elements ($v_0, ..., v_7$) we need to divide $p_0$ by the following value:
 
 $$
-\alpha_0 + \alpha_1 \cdot m_{bp} + \alpha_2 \cdot r + \alpha_4 \cdot 8 + \sum_{i=0}^7 (\alpha_{i+8} \cdot v_i)
+\alpha_0 + \alpha_1 \cdot m_{bp} + \alpha_2 \cdot r + f_{ctrlb} \cdot \alpha_5 \cdot \sum_{b=0}^6(b_i \cdot 2^i) + \sum_{i=0}^7 (\alpha_{i+8} \cdot v_i)
 $$
 
 where:
 * $m_{bp}$ is a label indicating beginning of a new permutation. Value of this label is computed based on hash chiplet selector flags according to the methodology described [here](../chiplets/hasher.md#multiset-check-constraints).
+* $f_{ctrlb}$ is a flag indicating whether the current operation is a control block initializer (excluding span).
```

Comment:
Should these references to $f_{ctrlb}$ be removed now?

Reply:
good catch, removed.